### ISSUE TITLE
Clarification: Does this depend on getting that pull request to polymer?

### ISSUE THREAD

**Issue by Analect** [PolymerEl/d3-layout-tree#2]
@christophe-g 
Thanks for this element.  Can you confirm if it will only work if the pull request (https://github.com/Polymer/polymer/pull/3372) you point to in the README gets merged?

Thanks

**Comment by christophe-g** [comment]
Indeed, I confirm it's only working with the pull request. But You can merge locally...

P.S. on leave for a few days, won't be able to provide support ; )
C.
